controller/orphan_controller.go

```diff
@@ -269,48 +303,195 @@ func (oc *OrphanController) isResponsibleFor(orphan *longhorn.Orphan) bool {
 	return isControllerResponsibleFor(oc.controllerID, oc.ds, orphan.Name, orphan.Spec.NodeID, orphan.Status.OwnerID)
 }
 
-func (oc *OrphanController) cleanupOrphanedData(orphan *longhorn.Orphan) (err error) {
+func (oc *OrphanController) cleanupOrphanedResource(orphan *longhorn.Orphan) (isCleanupComplete bool, err error) {
```

Comment:
I'm fine with the change. 
Resource can mean replica directory, replica instance, engine instance in the data plane.